ros2_ws/Dockerfile

```diff
@@ -35,7 +47,19 @@ RUN ( \
   && mkdir /run/sshd
 
 
-FROM base-dependencies as base-workspace
+FROM base-dependencies as python3-install
+ARG PYTHON_VER=3.10.2
+WORKDIR /tmp
+
+RUN wget https://www.python.org/ftp/python/${PYTHON_VER}/Python-${PYTHON_VER}.tar.xz
+RUN sudo tar -xf Python-${PYTHON_VER}.tar.xz
```

Comment:
I missed this before but why do you do `sudo` here? you are already root here

Reply:
ah it's true, good point. I was used to having derived docker images start as USER ros2, but of course this is the base image. I can address that as part of the fix in #29